lib/parser.es6

```diff
@@ -1,15 +1,16 @@
 class Parser {
 
-  constructor(template) {
+  constructor(template, defaultMember=0) {
```

Comment:
actually, perhaps make it a required argument as we'll always be passing it in? This will make the tests variants simpler too as we'd just modify `assertValidTemplate` to accept a required `emberNamespace` argument